docs/source/install/common/start_services.rst

```diff
@@ -2,6 +2,6 @@
 
     sudo st2ctl start
 
-* Register sensors and actions ::
+* Register sensors, rules and actions ::
```

Comment:
@Kami thanks for pointing out. If I check the "sudo st2ctl reload" output on st2 2.1.0, it says:

2017-01-17 08:47:25,560 INFO [-] =========================================================
2017-01-17 08:47:25,560 INFO [-] ############## Registering rules ########################
2017-01-17 08:47:25,560 INFO [-] =========================================================
2017-01-17 08:47:26,122 INFO [-] Registered 17 rules.

Am I missing something else?